core/src/main/java/de/bwaldvogel/mongo/bson/BsonTimestamp.java

```diff
@@ -15,6 +16,11 @@ public BsonTimestamp(Instant instant) {
         this(instant.toEpochMilli());
```

Comment:
Shouldn't time be epoch seconds ? 

> This internal timestamp type is a 64 bit value where:
> the most significant 32 bits are a time_t value (**seconds since the Unix epoch**)
> the least significant 32 bits are an incrementing ordinal for operations within a given second.